cockpit/cockpit/manifests/init.pp

```diff
@@ -0,0 +1,25 @@
+class cockpit {
+
+  package { 'cockpit':
+    ensure => latest,
+  }
+
+  $content =  "# This file is managed by puppet
```

Comment:
you might look into putting this in `files/cockpit.conf` and changing `content => $content`

to: `source => 'puppet://cockpit/cockpit.conf'`

this is the standard Puppet approach for static files

Reply:
When I use:

`source => 'puppet://cockpit/cockpit.conf'`

I get this error

```
Error: /Stage[main]/Cockpit/File[/etc/cockpit/cockpit.conf]: Could not evaluate: Could not retrieve file metadata for puppet://cockpit/cockpit.conf: Failed to open TCP connection to cockpit:8140 (getaddrinfo: Name or service not known)
```

I did manage to get it working with the absolute path instead

`source => '/root/cockpit/cockpit/files/cockpit.conf'`